wordpress-mobile/WordPress-iOS
# Login Epilogue: move connect site option

Ref #13413 

This moves the 'Connect another site' option.
- 'Connect another site' button removed.
- 'Connect another site' row added to the bottom of the sites list.
- The 'Continue' button has been renamed to 'Done'.

Not yet updated:
- Button area background.
- iPad specific layout.
- Accessibility.

---
To test:
- Log into the app.
- Verify 'Connect another site' appears at the bottom of the list, and launches the site connection flow.
- Verify 'Connect another site' looks like the Zeplin design, or that on #13413 .


|  ![short_list_light](https://user-images.githubusercontent.com/1816888/75389357-9ae78880-58a3-11ea-8343-e023856986f5.png) |  ![short_list_dark](https://user-images.githubusercontent.com/1816888/75389375-a1760000-58a3-11ea-9636-fbeafb5476a5.png) |
|--------|-------|
|  ![long_list_light](https://user-images.githubusercontent.com/1816888/75389423-b3f03980-58a3-11ea-8131-02ac34600c24.png) |  ![long_list_dark](https://user-images.githubusercontent.com/1816888/75389443-ba7eb100-58a3-11ea-9d04-f5958f660d3c.png) |
|  ![no_sites_light](https://user-images.githubusercontent.com/1816888/75390023-c8810180-58a4-11ea-9d45-df6259a10c96.png) |  ![no_sites_dark](https://user-images.githubusercontent.com/1816888/75390037-cfa80f80-58a4-11ea-9c1a-9528a37ff095.png) |
---
PR submission checklist:
